### PR TITLE
toggle: renamed toggle(showOrHide) to toggle(doShow)

### PR DESCRIPTION
**Pull request by burka**
To make it clear that passing true shows and passing false hides. Otherwise if you take showOrHide at its word, it would do nothing if you pass false and would call the regular toggle() if you pass true ;-)

### CODE REVIEW

File: entries/toggle.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
   </signature>
   <signature>
     <added>1.3</added>
-    <argument name="showOrHide" type="Boolean">
+    <argument name="doShow" type="Boolean">
```
Comment (gnarf) [inline entries/toggle.xml]: Not sure that `doShow` does this any better...  How about `setShown` ?  

Also maybe add a description in the `<desc>` tag specifying `true` will show and `false` will hide?

**Comment by gnarf** [comment]
Thanks for the submission! Sorry that my `showOrHide` name was confusing to :smile:

**Comment by burka** [comment]
Glad if it clears things up. At least for me ;-) `setShown` or `doShow` both sound like the initiation of a action, both are valid to me and I think both would make it more clear. I have no definite opinion on that, so setShown is ok for me if you prefer it.

**Comment by gnarf** [comment]
:+1: Looks great, thanks for the quick update!

**Comment by AurelioDeRosa** [comment]
IMHO `doShow` doesn't clarify the concept. `doShow` seems tied to the show concept only, and not that passing `false`  hides the elements.

**Comment by scottgonzalez** [comment]
How about `makeVisible` (or honestly, just leaving it as is).

**Comment by gnarf** [comment]
I think the change to the description is important either way, what we call it doesn't matter that much to me, any name for that parameter is awkward.

**Comment by AurelioDeRosa** [comment]
:+1: for leaving it as is

**Comment by burka** [comment]
The description should be changed in any case. Currently if you read it you might make the right guess, but you can't infer the meaning of the parameter from the documentation without reading the code snippet further down.

I think all alternates are a little awkward, but faster to understand than `showOrHide`. `makeVisible` might be a little better, just `show` might be a little better, `doShow`, `setShown` are both better than the current version and could be used until someone has a even better idea.

**Comment by gnarf** [comment]
I like it as you have it for now. Going to merge it.

**Comment by kswedberg** [comment]
Too bad. I agree that the description is better, but `setShown` is worse. It implies that the method will either show it or it won't, when what it really does is show it or hide it depending on its current state (hence the "showOrHide").

**Comment by AurelioDeRosa** [comment]
I agree with @kswedberg. It wasn't an urgent fix, so we could have discussed a bit more.

**Comment by gnarf** [comment]
I know we went back and forth on it when documenting it originally I think
'showOrHide' was my own suggestion, and I'm seeing problems with it now.
'set shown' felt better.  Let's talk it over in an issue and come back to
it if needed?

**Comment by gnarf** [comment]
If you have opinions on what we should call this, please go to #655 - Thanks @burka for the submission (and starting this discussion inadvertently)!

**Comment by gnarf** [comment]
Also @AurelioDeRosa I wasn't ignoring your "I liked it the old way" comment before the merge, I just didn't read it until *after* I had merged :frowning: sorry

**Comment by AurelioDeRosa** [comment]
Don't worry @gnarf, I wasn't complaining :)

**Pull request closed without merging**